packages/edit-site/src/components/list/index.js

```diff
@@ -51,7 +51,7 @@ export default function List() {
 		[ templateType ]
 	);
 
-	useTitle( postType?.labels?.name );
+	useTitle( postType, postType );
```

Comment:
Not sure why the first parameter has been changed?
I'm not even sure this `List` component is actually used anywhere though.

Reply:
Oversight, reverted.